pkg/virt-api/rest/expand.go

```diff
@@ -80,7 +80,7 @@ func (app *SubresourceAPIApp) ExpandSpecVMRequestHandler(request *restful.Reques
 }
 
 func (app *SubresourceAPIApp) expandSpecResponse(vm *v1.VirtualMachine, errorFunc func(error) *errors.StatusError, response *restful.Response) {
-	expandedVM, err := app.instancetypeMethods.Expand(vm, app.clusterConfig)
```

Comment:
Can we nuke `Expand` from `instancetypeMethods` yet?

Reply:
I think so. I'll have a look.